Apps/Admin/Client/Store/Communications/CommunicationsReducers.cs

```diff
@@ -161,6 +161,84 @@ public static CommunicationsState ReduceAddFailAction(CommunicationsState state,
         };
     }
 
+    /// <summary>
+    /// The reducer for updating communications.
+    /// </summary>
+    /// <param name="state">The communications state.</param>
+    /// <returns>The new state.</returns>
+    [ReducerMethod(typeof(CommunicationsActions.UpdateAction))]
+    public static CommunicationsState ReduceUpdateAction(CommunicationsState state)
+    {
+        return state with
+        {
+            Update = state.Update with
+            {
+                IsLoading = true,
+            },
+            IsLoading = true,
```

Comment:
why two IsLoading?

Reply:
For the communications state, I experimented with tracking both the state of individual requests (Load, Add, Update, Delete) as well as an overall state, so consumers can check if any requests are loading or have encountered an error, or if a specific request is loading or has encountered an error.

I think I ended up retrieving the global `IsLoading` state along with individual `Error` states.